PolyEngine/Core/Core.vcxproj

```diff
@@ -130,9 +130,7 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="Src\BaseObject.cpp" />
     <ClCompile Include="Src\Color.cpp" />
-    <ClCompile Include="Src\CorePCH.cpp">
```

Comment:
Well in MSVC they do. http://www.asawicki.info/productions/artykuly/PrecompiledHeaders.php5